contracts/exchange-wrappers/Starkware.sol

```diff
@@ -0,0 +1,39 @@
+/*
+    Copyright 2021 dYdX Trading Inc.
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+pragma solidity ^0.5.5;
```

Comment:
Why use `0.5.5` instead of a newer solidity version?

Reply:
the open zeplin requires it, I can see if there's a newer package version